docstore/mongodocstore/localmongo.sh

```diff
@@ -13,14 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Starts a local MongoDB instance via Docker.
+# Starts two local MongoDB instances (v3 and v4) via Docker.
 
 # https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
 set -euo pipefail
 
-echo "Starting MongoDB..."
-docker rm -f mongo &> /dev/null || :
-docker run -d --name mongo  -p 27017:27017 mongo:4 &> /dev/null
-echo "...done. Run \"docker rm -f mongo\" to clean up the container."
+echo "Starting MongoDB v4 listening on 27017..."
+docker rm -f mongo4 &> /dev/null || :
+docker run -d --name mongo4  -p 27017:27017 mongo:4 &> /dev/null
+echo "...done. Run \"docker rm -f mongo4\" to clean up the container."
+echo
+
+echo "Starting MongoDB v3 listening on 27018..."
+docker rm -f mongo3 &> /dev/null || :
+docker run -d --name mongo3  -p 27018:27017 mongo:3 &> /dev/null
```

Comment:
From docker docs, I see that this means port 27018 of the host is mapped to port 27017 of the container. So does this mean both mongos will map to the same container? Might be a good idea to have the container have a unique number other than the same as the port number for one of mongos.

Reply:
There are two containers differentiated by the tags: `mongo:3` vs `mongo:4`, so this maps the 27017 port of mongo:4 to the host's 27017 and the 27017 port of mongo:3 to the host's 27018.

Reply:
It looked like a copy and paste mistake to me because I did find the host ports in the files below, but not the mongo's to confirm. So maybe add a comment at least?

Reply:
I think the `echo "Starting MongoDB ..."` above serves the purpose already. I clarified a bit at the top-level comment.